src/index.ts

```diff
@@ -29,10 +43,28 @@ export function runInContext(
   context: Context,
   options: Partial<IOptions> = {}
 ): Promise<Result> {
+  function getFirstLine(theProgram: Program) {
+    const firstLineOfProgram = theProgram.body[0] as ExpressionStatement
+    if (!!firstLineOfProgram) {
```

Comment:
I think it's better if you explicitly check if the first line exists, then check if the first line type === 'ExpressionStatement'.